File: sdk/nim/src/microsandbox.nim
## Microsandbox Nim SDK
## A minimal SDK for the Microsandbox project.

proc greet*(name: string): string =
  ## Returns a greeting message for the given name.
  ##
  ## Example:
  ##   let message = greet("World")
  ##   echo message
  let message = "Hello, " & name & "! Welcome to Microsandbox!"
  echo message
  return message

when isMainModule:
  # Simple test
  let result = greet("Test")
  assert "Hello, Test!" in result
  echo "Test passed!"

File: sdk/ruby/lib/microsandbox.rb
# frozen_string_literal: true

require "microsandbox/version"

# Microsandbox SDK for Ruby
module Microsandbox
  class Error < StandardError; end

  # Returns a greeting message for the given name
  #
  # @param name [String] The name to greet
  # @return [String] A greeting message
  def self.greet(name)
    message = "Hello, #{name}! Welcome to Microsandbox!"
    puts message
    message
  end
end
